static/data/2017/11/events.json

```diff
@@ -1,12 +1,12 @@
 [
   {
-    "name": "AR/VR Hackathon",
```

Comment:
@Toyrone It seems like the PR is removing `AR/VR Hackathon`. 
Please follow these steps:
- Reset to your original commit
- Run `git pull --rebase upstream master`
- Run `git mergetool --tool=emerge`
- Resolve the conflicts